apps/back-office/src/app/melding/[meldingId]/_components/BackLink.tsx

```diff
@@ -0,0 +1,25 @@
+'use client'
+
+import { Icon, Link } from '@amsterdam/design-system-react'
+import { ChevronLeftIcon } from '@amsterdam/design-system-react-icons'
+import NextLink from 'next/link'
+import type { AnchorHTMLAttributes } from 'react'
+
+import styles from './BackLink.module.css'
+
+type Props = Omit<AnchorHTMLAttributes<HTMLAnchorElement>, 'color'> & {
+  href: string
+}
+
+export const BackLink = ({ children, className = '', href, ...restProps }: Props) => {
+  const combinedClassName = `${styles.link} ${className}`.trim()
+
+  return (
+    <NextLink href={href} legacyBehavior passHref>
+      <Link {...restProps} className={combinedClassName}>
+        <Icon svg={ChevronLeftIcon} size="level-5" />
+        {children}
+      </Link>
+    </NextLink>
+  )
+}
```

Comment:
Should this component live in libs/ui since it an exact copy of the component in publuc?

Reply:
I thought about it, but finally decided against it, for 2 reasons:

- [The rule of three](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming))
- Our internal UI library currently does not have any Next specific stuff. 2 of our apps are Next apps, but 1 is not. If we were to add Next specific stuff to our UI lib, it wouldn't be a generic UI library anymore.

Reply:
oke, I see the point. Let's stick to this for now then.